spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -208,6 +207,8 @@ def handle_itc_response(raw, flaky_api_call: false)
           raise_insuffient_permission_error!
         elsif flaky_api_call
           raise ITunesConnectPotentialServerError.new, errors.join(' ')
+        elsif errors.count == 1 and errors.first.first.include?("Problem processing review submission.")
```

Comment:
Did you mean `errors.first.first`? The other checks just use `errors.first`. If `errors.first` is a `String` you will get `NoMethodError`

Reply:
I tested this with mock flow. What kind of change are you expecting me to add?

Reply:
Yes, can confirm that this will eventually crash with `NoMethodError`, as you don't know how many errors are there, could be an empty array